### PR TITLE
LPS-137541 Normalize value to prevent issues when pasting hexcode

### PR DESCRIPTION
**Pull request by fortunatomaldonado**
https://issues.liferay.com/browse/LPS-137541

The issue is pasting a hexcode with `#` in the color picker for a widget, will not register the color change. This happens since the value after the paste will be `##e05929`. The input does not have a validation so it will not display an error.

The way I was able to solve this is to reintroduce `normalizeColor` and have it remove all `#`s. I am using the `colorValue` to create the new value. I tried normalizing the `color`, but I kept seeing the `#` when testing. When I created `normalizeColorValue` I was then able to see the color working correctly.

Let me know if there are any questions or comments about this.
Thank you.

### CODE REVIEW

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

File: modules/apps/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ColorPickerInput.es.js [hunk 1]
```diff
@@ -15,12 +15,18 @@
...
+	return colorValue.replace('#', '');
+}
+
 const HEX_COLOR_REGEX = /^#?[0-9A-F]{3}(?:[0-9A-F]{3})?$/i;
 
 const ColorPicker = ({color, label, name}) => {
 	const [colorValue, setColorValue] = useState(color);
 	const [customColors, setCustomColors] = useState([]);
 
+	const normalizeColorValue = normalizeColor(colorValue);
```
Comment (bryceosterhaus) [inline modules/apps/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ColorPickerInput.es.js:28]: Not a blocker, but there really isn't a need to extract `normalizeColor` into its own function. You could simply just inline the `.replace`. And I would also rename the variable to be a bit more explicit

```suggestion
	const noHashColorValue = colorValue.replace('#', '');
```

Comment (fortunatomaldonado) [reply modules/apps/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ColorPickerInput.es.js]: Truly appreciate the feedback! Made the changes.

**Comment by bryceosterhaus** [comment]
ci:forward

**Pull request closed without merging**